ci/setup-build-test-gdk.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+source /opt/improbable/environment
```

Comment:
Wrap in an if-exists check since you don't have the standard be-a-pedant bash mode set `set -o errexit` `set -o nounset` `set -o pipefail`

Reply:
adding 
```set -e -u -o pipefail
if [[ -n "${DEBUG-}" ]]; then
    set -x
fi```

to all the bash files right now, so that should fix this